### PR TITLE
RavenDB-10034 SQL ETL - Support additional databases 

### PR DESCRIPTION
**Pull request by arekpalinski**

### CODE REVIEW

**Review by ppekrol — APPROVED** [review]

**Comment by ppekrol** [comment]
I'm wondering at what other DBs we should look at?

File: src/Raven.Client/Documents/Operations/ETL/SQL/SqlConnectionStringParser.cs [hunk 1]
```diff
@@ -24,11 +23,70 @@ internal static (string Database, string Server) GetDatabaseAndServerFromConnect
...
 
-                    var port = GetConnectionStringValue(connectionString, new[] { "Port" });
+                    var postgrePort = GetConnectionStringValue(connectionString, new[] { "Port" });
 
-                    if (string.IsNullOrEmpty(port))
-                        server += $":{port}";
+                    if (string.IsNullOrEmpty(postgrePort))
+                        server += $":{postgrePort}";
                     break;
+                case SqlProvider.MySqlClient:
```
Comment (ayende) [inline src/Raven.Client/Documents/Operations/ETL/SQL/SqlConnectionStringParser.cs:31]: Do we really need to parse the connection strings? Why not pass them as is to the `AdoConnection`?

**Comment by ayende** [comment]
I think that this close the loop in terms of what we support until  we get a user request.
I don't think we want to pre-preemptively support DB2, for example.

**Comment by ayende** [comment]
I left my mainframe on my other pants

File: src/Raven.Client/Documents/Operations/ETL/SQL/SqlConnectionStringParser.cs (hunk 1 above)
Comment (arekpalinski) [reply src/Raven.Client/Documents/Operations/ETL/SQL/SqlConnectionStringParser.cs:31]: This class (and its friends that use it) are in Raven.Client where we don't have ADO dependencies. The values that we parse here are displayed in the studio (ongoing task view) and go into the logs (`dbname@server`).  When we do the actual connection then we pass the entire connection string.

**Pull request merged**